sklearn/metrics/tests/test_classification.py

```diff
@@ -303,6 +303,18 @@ def test_precision_recall_f1_score_multiclass():
     assert_array_equal(s, [24, 20, 31])
 
 
+def test_precision_refcall_f1_score_multilabel_unordered_labels():
```

Comment:
Might as well test this in the multiclass case and for different averaging strategies, too, no?